gui/control.js

```diff
@@ -43,14 +48,15 @@ function render() {
         accountsDiv.insertAdjacentHTML('beforeend', userCardString);
 
         // Deleting User Profile
-        document.getElementById(`switch-btn-${nickname}`).addEventListener('click', async function () {
-            logger.debug(`switching ${nickname}`)
-            const error = await PSS.switchAccount(nickname);
-            if (!error) return;
-            showToast("Error!", errorToMessage[error])
-        });
+        document.getElementById(`switch-btn-${nickname}`).addEventListener('click', async function(){
+            await switchAccountHandler(nickname);
+        }); 
 
-        document.getElementById(`delete-btn-${nickname}`).addEventListener('click', async function () {
+        document.getElementById(`nick-${nickname}`).addEventListener('dblclick', async function () {
+            await switchAccountHandler(nickname);
+        });
```

Comment:
use arrow function instead, for better readability and why are we using async/await here?